github/orgs_members_test.go

```diff
@@ -395,7 +395,11 @@ func TestOrganizationsService_ListPendingOrgInvitations(t *testing.T) {
 	})
 
 	opt := &ListOptions{Page: 1}
+<<<<<<< HEAD
 	invitations, _, err := client.Organizations.ListPendingOrgInvitations(context.Background(), 1, opt)
+=======
+	invitations, _, err := client.Organizations.ListPendingOrgInvitations(1, opt)
+>>>>>>> 1bd4328a520dfc6d58f92e0f441f3f56718b725d
```

Comment:
@shurcooL this was part of messed up fork of mine. I have corrected it and now the latest check in has the right changes.